src/components/Combobox.tsx

```diff
@@ -0,0 +1,130 @@
+import { Menu, MenuItem, Popover, PopoverPanel } from 'solid-headless'
+import { Accessor, Setter, For, createEffect, createMemo, createSignal, onCleanup } from 'solid-js'
+import { FiExternalLink } from 'solid-icons/fi'
+import { FaSolidCheck } from 'solid-icons/fa'
+import { selectedComboboxItems, setSelectedComboboxItems } from '../FilterStore'
+export interface comboboxItem {
+	name: string
+	eventId: string
+}
+
+
+export const Combobox = () => {
+	const [panelOpen, sePanelOpen] = createSignal(false)
+	const [usingPanel, setUsingPanel] = createSignal(false)
+	const [inputValue, setInputValue] = createSignal('')
+	const [comboboxItems, setComboboxItems] = createSignal<comboboxItem[]>([
+		{ name: 'Policy', eventId: 'policy' },
+		{ name: 'LD', eventId: 'ld' }
```

Comment:
We should be either defining these above line 12 as a static object or pass them as a prop. 

I would prefer to define them as a static object though.